core/server/config/utils.js

```diff
@@ -37,7 +37,8 @@ exports.makePathsAbsolute = function makePathsAbsolute(obj, parent) {
         } else if (
             _.isString(configValue) &&
             (configValue.match(/\/+|\\+/) || configValue === '.') &&
-            !path.isAbsolute(configValue)
+            !path.isAbsolute(configValue) &&
+            (!configValue.match('-----BEGIN') || !configValue.match('-----END'))
```

Comment:
So this is definitely too specific. What we need here is a check to figure out if a string looks like a path.

e.g. we can create the absolute path and then check if the value is a valid absolute path. If not, we fallback to the original value. If you google `regex path validation` - there are common regex patterns we can use to check if the absolute path looks like a path. 

If you have time, it would be also good to extend our tests a bit. But i can also push some after you have changed this :)

Reply:
I followed the regex suggestion and have updated the PR. Thanks again for the feedback and review.